recipes/NCEPLIBS-bufr/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "NCEPLIBS-bufr" %}
+{% set version = "12.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-EMC/NCEPLIBS-bufr/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 525f26238dba6511a453fc71cecc05f59e4800a603de2abbbbfb8cbb5adf5708
+
+build:
+  # NOTE: No Windows support in initial feedstock (known use cases are only osx and linux)
+  skip: True  # [win]
+  # NOTE: Only support recent Python releases. Cannot support Python==3.12 because build
+  # system in NCEPLIBS-bufr_v12.0.1 uses a hard-coded dependency on distutils, which was
```

Comment:
Fortran extensions are in a pickle with py312 :-/
They need to move to cmake or meson and that is not an easy change. Let me know if you figure out a path with least friction to solve this!

Reply:
The good news is that this particular package does have an improvement to its cmake-based build already merged into trunk, so the next time they cut a release we'll be able to support Py312. But yeah, wish there was a generic way (maybe a back-port of distutils?)